content/04.discussion.md

```diff
@@ -1,26 +1,31 @@
-Furthermore, in the absence of DNA methylation data, quick molecular subtyping classification could not be performed.
-Thus, to enable more targeted analyses to interrogate the biology of specific tumor subtypes, we created subtyping modules aligned with WHO molecularly-defined diagnoses.
-We worked closely with pathologists and clinicians to build modules from which we determined a research-grade integrated diagnosis for 44% (1260/2840) of biospecimens as well as identify incorrectly diagnosed or mis-identified samples in the OpenPBTA cohort. 
+Furthermore, in the absence of DNA methylation data, rapid classification to derive molecular subtypes could not be performed.
+Thus, to enable biological interrogation of specific tumor subtypes, we created RNA- and DNA-based subtyping modules aligned with WHO molecularly-defined diagnoses.
+We worked closely with pathologists and clinicians to build modules from which we determined a research-grade integrated diagnosis for 44% (1260/2840) of biospecimens as well as discovered incorrectly diagnosed or mis-identified samples in the OpenPBTA cohort. 
 
-We applied multiple previously validated classifiers to RNA expression data.
-We predicted medulloblastoma subtypes [@doi:10.1186/s13029-016-0053-y; @doi:10.1371/journal.pcbi.1008263], whether tumors have dysfunctional _TP53_ [@doi:10.1016/j.celrep.2018.03.076], and their degree of telomerase activity [@doi:10.1038/s41467-020-20474-9].
-By assessing _TP53_ and telomerase activity prospectively from expression data, information previously only attainable with genomic sequencing and/or qPCR, we can quickly add knowledge to our existing biological understanding of these tumors.
+We harnessed RNA expression data for a number of analyses, including application of multiple previously validated classifiers.
+From 43/122 (35.2%) medulloblastoma tumors whose pathology reports contained subtype information, we accurately predicted subtypes using MM2S (90.7%) or medulloPackage (95.4%) [@doi:10.1186/s13029-016-0053-y; @doi:10.1371/journal.pcbi.1008263].
+We used additional classifiers to determine whether tumors have dysfunctional _TP53_ [@doi:10.1016/j.celrep.2018.03.076] and their degree of telomerase activity from EXTEND scores [@doi:10.1038/s41467-020-20474-9].
+We found that hypermutant tumors universally display dysregulation of _TP53_ and that EXTEND scores are a robust surrogate measure for telomerase activity, as tumors with high scores have a significant adverse risk for overall survival. 
+Additionally, high _TP53_ scores are a significant negative prognostic indicator for overall survival for patients with certain tumor types, such as H3 K28-altered DMGs and ependymomas. 
+By assessing _TP53_ and telomerase activity prospectively from expression data, information previously only attainable with DNA sequencing and/or qPCR, we can quickly add oncogenic biomarker and prognostic knowledge to our existing biological understanding of these tumors.
```

Comment:
```suggestion
By assessing _TP53_ and telomerase activity prospectively from expression data, which is usually only attainable with DNA sequencing and/or qPCR, we incorporated oncogenic biomarker and prognostic knowledge that expanded our biological understanding of these tumors.
```

I'm not sure I followed this sentence. I tried to rephrase to follow it better, but I may or may not have got it right?

Reply:
yes, but kept active, as we want to relay that this is actively useful to do at diagnosis/time of biopsy